devsetup/tripleo/initial-ceph.conf

```diff
@@ -0,0 +1,2 @@
+[global]
+osd_pool_default_size=1
```

Comment:
This represents a workaround for a 3+1 scenario.
I'm ok with the rest of the patch but I'm wondering if we want to merge the workaround, or hold this patch until a 3+3 topology (that doesn't need this) is in place.